Feature/add ticket button

jspengeman opened the pull request:
A user can now add a ticket to a specific project that they are currently viewing.

Commit:
Add ticket button now working

Commit:
Add ticket button now implemented

Commit:
Styled the add ticket page

alexfresnostate commented:
I see s lot of effort from you today. great job.

Have a break and happy thanksgiving.
On Nov 27, 2014 1:40 PM, "jspengeman" notifications@github.com wrote:

> A user can now add a ticket to a specific project that they are currently
> 
> ## viewing.
> 
> You can merge this Pull Request by running
> 
>   git pull https://github.com/implacable/projectManager feature/AddTicketButton
> 
> Or view, comment on, or merge it at:
> 
>   https://github.com/implacable/projectManager/pull/30
> Commit Summary
> - Add ticket button now working
> - Add ticket button now implemented
> - Styled the add ticket page
> 
> File Changes
> - _M_ project_ticket/forms.py
>   https://github.com/implacable/projectManager/pull/30/files#diff-0
>   (1)
> - _A_ project_ticket/static/project_ticket/css/add_ticket_style.css
>   https://github.com/implacable/projectManager/pull/30/files#diff-1
>   (14)
> - _M_ project_ticket/static/project_ticket/css/project_style.css
>   https://github.com/implacable/projectManager/pull/30/files#diff-2
>   (21)
> - _M_ project_ticket/templates/project_ticket/addticket.html
>   https://github.com/implacable/projectManager/pull/30/files#diff-3
>   (94)
> - _M_ project_ticket/templates/project_ticket/project.html
>   https://github.com/implacable/projectManager/pull/30/files#diff-4
>   (11)
> - _M_ project_ticket/urls.py
>   https://github.com/implacable/projectManager/pull/30/files#diff-5
>   (2)
> - _M_ project_ticket/views.py
>   https://github.com/implacable/projectManager/pull/30/files#diff-6
>   (8)
> 
> Patch Links:
> - https://github.com/implacable/projectManager/pull/30.patch
> - https://github.com/implacable/projectManager/pull/30.diff
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/implacable/projectManager/pull/30.

jspengeman commented:
Thank you for noticing Alex. I really appreciate you noticing. I added
about 1000 lines of code today and yesterday. I am glad someone took
notice. Once again, thank you.

ps: I got your other email and I don't really have any questions on the
project right now.
- Jon

On Thursday, November 27, 2014, alexfresnostate notifications@github.com
wrote:

> I see s lot of effort from you today. great job.
> 
> Have a break and happy thanksgiving.
> On Nov 27, 2014 1:40 PM, "jspengeman" <notifications@github.com
> <javascript:_e(%7B%7D,'cvml','notifications@github.com');>> wrote:
> 
> > A user can now add a ticket to a specific project that they are
> > currently
> > 
> > ## viewing.
> > 
> > You can merge this Pull Request by running
> > 
> > git pull https://github.com/implacable/projectManager
> > feature/AddTicketButton
> > 
> > Or view, comment on, or merge it at:
> > 
> > https://github.com/implacable/projectManager/pull/30
> > Commit Summary
> > - Add ticket button now working
> > - Add ticket button now implemented
> > - Styled the add ticket page
> > 
> > File Changes
> > - _M_ project_ticket/forms.py
> >   https://github.com/implacable/projectManager/pull/30/files#diff-0
> >   (1)
> > - _A_ project_ticket/static/project_ticket/css/add_ticket_style.css
> >   https://github.com/implacable/projectManager/pull/30/files#diff-1
> >   (14)
> > - _M_ project_ticket/static/project_ticket/css/project_style.css
> >   https://github.com/implacable/projectManager/pull/30/files#diff-2
> >   (21)
> > - _M_ project_ticket/templates/project_ticket/addticket.html
> >   https://github.com/implacable/projectManager/pull/30/files#diff-3
> >   (94)
> > - _M_ project_ticket/templates/project_ticket/project.html
> >   https://github.com/implacable/projectManager/pull/30/files#diff-4
> >   (11)
> > - _M_ project_ticket/urls.py
> >   https://github.com/implacable/projectManager/pull/30/files#diff-5
> >   (2)
> > - _M_ project_ticket/views.py
> >   https://github.com/implacable/projectManager/pull/30/files#diff-6
> >   (8)
> > 
> > Patch Links:
> > - https://github.com/implacable/projectManager/pull/30.patch
> > - https://github.com/implacable/projectManager/pull/30.diff
> > 
> > —
> > Reply to this email directly or view it on GitHub
> > https://github.com/implacable/projectManager/pull/30.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/implacable/projectManager/pull/30#issuecomment-64840470
> .

alexfresnostate commented:
It is really awesome. Thanks for being so dedicate to your team and
project. Hope you learn and earn Something for your career.

As for 150 team part you are definitely super.
On Nov 27, 2014 6:53 PM, "jspengeman" notifications@github.com wrote:

> Thank you for noticing Alex. I really appreciate you noticing. I added
> about 1000 lines of code today and yesterday. I am glad someone took
> notice. Once again, thank you.
> 
> ps: I got your other email and I don't really have any questions on the
> project right now.
> - Jon
> 
> On Thursday, November 27, 2014, alexfresnostate notifications@github.com
> 
> wrote:
> 
> > I see s lot of effort from you today. great job.
> > 
> > Have a break and happy thanksgiving.
> > On Nov 27, 2014 1:40 PM, "jspengeman" <notifications@github.com
> > <javascript:_e(%7B%7D,'cvml','notifications@github.com');>> wrote:
> > 
> > > A user can now add a ticket to a specific project that they are
> > > currently
> > > 
> > > ## viewing.
> > > 
> > > You can merge this Pull Request by running
> > > 
> > > git pull https://github.com/implacable/projectManager
> > > feature/AddTicketButton
> > > 
> > > Or view, comment on, or merge it at:
> > > 
> > > https://github.com/implacable/projectManager/pull/30
> > > Commit Summary
> > > - Add ticket button now working
> > > - Add ticket button now implemented
> > > - Styled the add ticket page
> > > 
> > > File Changes
> > > - _M_ project_ticket/forms.py
> > >   https://github.com/implacable/projectManager/pull/30/files#diff-0
> > >   (1)
> > > - _A_ project_ticket/static/project_ticket/css/add_ticket_style.css
> > >   https://github.com/implacable/projectManager/pull/30/files#diff-1
> > >   (14)
> > > - _M_ project_ticket/static/project_ticket/css/project_style.css
> > >   https://github.com/implacable/projectManager/pull/30/files#diff-2
> > >   (21)
> > > - _M_ project_ticket/templates/project_ticket/addticket.html
> > >   https://github.com/implacable/projectManager/pull/30/files#diff-3
> > >   (94)
> > > - _M_ project_ticket/templates/project_ticket/project.html
> > >   https://github.com/implacable/projectManager/pull/30/files#diff-4
> > >   (11)
> > > - _M_ project_ticket/urls.py
> > >   https://github.com/implacable/projectManager/pull/30/files#diff-5
> > >   (2)
> > > - _M_ project_ticket/views.py
> > >   https://github.com/implacable/projectManager/pull/30/files#diff-6
> > >   (8)
> > > 
> > > Patch Links:
> > > - https://github.com/implacable/projectManager/pull/30.patch
> > > - https://github.com/implacable/projectManager/pull/30.diff
> > > 
> > > —
> > > Reply to this email directly or view it on GitHub
> > > https://github.com/implacable/projectManager/pull/30.
> > 
> > —
> > Reply to this email directly or view it on GitHub
> > <
> > https://github.com/implacable/projectManager/pull/30#issuecomment-64840470>
> > 
> > .
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/implacable/projectManager/pull/30#issuecomment-64847726
> .
